mod_customized/controllers.py

```diff
@@ -56,7 +57,10 @@ def index():
         if len(commit_arr) > 0:
             fork_test_form.commit_select.choices = commit_arr
             commit_options = True
+        fork_test_form.regression_test.choices = [(str(regression_test.id), regression_test)
```

Comment:
If you properly type the field, you can just pass int values:

https://wtforms.readthedocs.io/en/stable/fields.html#wtforms.fields.SelectField (the coerce)